SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DAQmx/AnalogOutput.cs

```diff
@@ -19,6 +19,7 @@ public static void WriteAnalogSingleSample(this DAQmxTasksBundle tasksBundle, do
         {
             tasksBundle.Do(taskInfo =>
             {
+                taskInfo.Task.Stream.ChannelsToRead = taskInfo.ChannelList;
```

Comment:
Analog output channels are used to do write operations. I think we should modify ChannelsToWrite instead.

Reply:
I had a check and noticed there is no ChannelsToWrite actually. We use a different mechanism to do the writes, which won't be impacted in similar scenarios.

So nothing should be done to Analog/DigitalOutput cs files.

Reply:
Removed the changes from Analog/Digital output files